docs/networking/networking-basics.md

```diff
@@ -0,0 +1,520 @@
+# Networking Basics
+
+> Written by Dan Ford and originally published on <a href="https://dlford.io/computer-networking-binary-hex-explained/" target="_blank" rel="noopener noreferrer">dlford.io</a>.<br />
+> Copyright &copy; 2019 dlford.io all rights reserved.
+
+This article is written with the intent of providing a basic understanding of computer networking and IP addressing concepts, which require a basic understanding of binary and hex. I've done my best to make it easy to follow for somebody with no experience on this topic. 
+
+_Note: I use the term "modem" very loosely in this article because most consumer modems these days have a built in switch and routing functions. You should be aware that technically a modem does not do routing, in only modulates a signal out to your ISP. In the context of this article I refer to a modem as a modem/router combo unit, when I say "router or modem", I am referring to whichever routing device your system is directly connected to._
+
+## Basic Network Settings
+
+### Discover Your Network
+
+In the context of a home network, you most likely have a modem or router handling DHCP, DNS, and routing for your network. The first thing you need to do is find the IP address of your router or modem on the network. An IP address is a string of numbers separated by decimal points, that identifies each computer on a network, and must therefore be unique to each computer on the same network.
+
+- Windows
+  - Right-click the start button, and choose Windows PowerShell
+  - Type the command `ipconfig | findstr /i "Gateway"`
+- Mac
+  - Go to Applications > Utilities, and double click on Terminal
+  - Type the command `netstat -nr | grep default`
+- Linux
+  - Open a terminal shell, the method for this varies by distribution, on Ubuntu you can use the hotkey `control` + `shift` + `t`
+  - Type the command `ip route | grep default`
+
+You should get a value of something similar to `192.168.0.1`, this is the IP address of your gateway. Open a web browser and put this number in the address bar with `http://` in front of it, for example `http://192.168.0.1`
+
+You will need to log in to this device, if you don't have the username and password, it may be on a sticker on the bottom of the unit (you should also change this after logging in to make your network more secure!). If it's not on a sticker, you'll have to look in the user manual for your modem or router, or search the web for "_(model number of your modem or router) default password_".
+
```

Comment:
There should be a big warning here that tells you that if you are using the default password you should change it